cmd/containerd-nydus-grpc/pkg/command/flags.go

```diff
@@ -209,6 +216,12 @@ func buildFlags(args *Args) []cli.Flag {
 			Usage:       "path to the kubeconfig file",
 			Destination: &args.KubeconfigPath,
 		},
+		&cli.BoolFlag{
+			Name:        "log-rotate",
```

Comment:
name it  `rotate-log` would be better

Reply:
Maybe change this to `log-rotate-max-size`, 0 means disable rotation?

Reply:
But there are still other rotation schemes.

Reply:
I think it's more practical to use `log-rotate-max-size` instead of `log-rotate` if we limit the number of CLI flags. In addition, we need to tell the user that it works only if the `--log-dir` is specified.

Reply:
Consider the case that the log's maximum size is set to 10GB while the backup count is set to 2. Then the log will be rotated without reaching size 10GB, which is against the user's parameter provided. In order to keep the parameter semantics consistent, log-rotate-max-size is not suitable until our nydus-snapshotter' s configuration file is composed